### DISCUSSION TITLE
Support for themes

### DISCUSSION THREAD

**Discussion by yaelgoede** [avisi-cloud/structurizr-site-generatr#306] [Q&A] [answered]
Hi i'm wondering if this solution wil also support the use of themes in the future. Right now the docker container cannot render a view of my workspace which uses the azure theme. 

Let me know, Thank You

**Comment by dirkgroot** [comment, marked as answer]
Hi @yaelgoede, themes are supported, to some extent. This program uses Structurizr's [C4 PlantUML Exporter](https://docs.structurizr.com/export/plantuml#c4plantumlexporter).

If you want to make use of themes or styles defined in your DSL, you will need to set the property `c4plantuml.tags` to `true`.

Issue #163 also contains some useful information about this (and the Azure theme in particular).

**Reply by yaelgoede** [reply]
NICE :D, this is perfect. Thank you
